kuttl-tests/Makefile

```diff
@@ -11,44 +12,55 @@ endif
 
 export PATH := $(shell pwd)/bin:$(PATH)
 export ROOT_DIR=$(realpath -L ./..)
+export DOCKER_API_VERSION=1.39
```

Comment:
I guess we're using the docker host from the AMI/Build Image? At least i had to force the version down to 1.39. Otherwise we'd use a client that defaults to 1.40, which fails